test/integration/queue/index.spec.ts

```diff
@@ -348,7 +343,7 @@ describe('queue', () => {
 
 			await expect(() => {
 				return ctx.flush(ctx.session);
-			}).rejects.toThrowError(errors.WorkerInvalidAction);
+			}).rejects.toThrowError();
```

Comment:
Was the `errors.WorkerInvalidAction` replaced by something else?

Reply:
`errors.WorkerInvalidAction` is still being used, but its being used within the native `assert.strict` meaning that what jest receives is actually an instance of an `AssertionError`. Here's what happens when I run that test expecting an `errors.WorkerInvalidAction` error:
```
    Expected constructor: WorkerInvalidAction
    Received constructor: AssertionError

    Received message: "WorkerInvalidAction: No such action: action-foo-bar@1.0.0"
```